### ISSUE TITLE
Need to update the specification based on latest notification support

### ISSUE THREAD

**Issue by Ziabatcha** [openBackhaul/MicroWaveDeviceInventory#801]
Need to update the specification based on the current implementation of device notifications. Because of old specification, notification is expecting resource attribute but that attribute is not supported by notification now. 

In the specification, https://github.com/openBackhaul/MicroWaveDeviceInventory/blob/develop/spec/MicroWaveDeviceInventory.yaml#L1144
```
   required:
   - counter
   - timestamp
  - resource
 - attribute-name
 - new-value
```
but resource attribute is changed to object path

**Comment by Ziabatcha** [comment]
Adding the mail conversation 
[mail_conversation.pdf](https://github.com/openBackhaul/MicroWaveDeviceInventory/files/13917782/mail_conversation.pdf)

**Comment by kmohr-soprasteria** [comment]
also see #803

**Comment by kmohr-soprasteria** [comment]
will be closed together with #803
